### PR TITLE
Add Gitter badge to README

### PR DESCRIPTION
**Pull request by jawshooah**
A fixed-up version of #13781.

[Review on Reviewable](https://reviewable.io/reviews/caskroom/homebrew-cask/13808)

### CODE REVIEW

**Comment by jawshooah** [comment]
@vitorgalvao The default Gitter badge doesn't fit the shields.io theme, so this is hard-coded for now (hopefully this will [change soon](https://github.com/badges/shields/issues/354)).

How do you feel about the color? Should we go with `green` or `brightgreen`?

**Comment by vitorgalvao** [comment]
`brightgreen` looks fine to me. That said, why not go with [blue](https://img.shields.io/badge/gitter-join%20chat-blue.svg)? I say this because variants of `red` / `yellow` / `green` tend to be taken in those badges to signify different build status and similar options.

Blue (or any colour other than the aforementioned three) is more neutral in that sense.

**Comment by vitorgalvao** [comment]
`brightgreen` looks fine to me. That said, why not go with [blue](https://img.shields.io/badge/gitter-join%20chat-blue.svg)? I say this because variants of `red` / `yellow` / `green` tend to be taken in those badges to signify different build status and similar options on a spectrum, pertaining to the code itself.

Blue (or any colour other than the aforementioned three) is more neutral in that sense.

**Comment by jawshooah** [comment]
I agree, the blue helps it stand out against the more code-centric badges. Fixed!

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ called `cask`.
 [![Build Status](https://img.shields.io/travis/caskroom/homebrew-cask/master.svg)](https://travis-ci.org/caskroom/homebrew-cask)
 [![Code Climate](https://img.shields.io/codeclimate/github/caskroom/homebrew-cask.svg)](https://codeclimate.com/github/caskroom/homebrew-cask)
 [![Coverage Status](https://img.shields.io/coveralls/caskroom/homebrew-cask.svg)](https://coveralls.io/r/caskroom/homebrew-cask)
+[![Join the chat at https://gitter.im/caskroom/homebrew-cask](https://img.shields.io/badge/gitter-join%20chat-blue.svg)](https://gitter.im/caskroom/homebrew-cask?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (vitorgalvao) [inline README.md]: Just read carefully the full url. Any reason we’re keeping `?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge`. I generally don’t like to give websites that information and hate what it does to the url; why keep it in this case?

Comment (jawshooah) [inline README.md]: That was the URL in the original PR, and I left it unchanged. I have no problem removing it, though. It's probably just to help Gitter collect data on how their service is being used.

Comment (vitorgalvao) [inline README.md]: If it is the same to you, I’d rather take that out. Apart from that, looks OK to merge.

**Comment by jawshooah** [comment]
Merged as 3bd694eda2a768a5ef1b769eb2a9365ef9aa2d04.

**Pull request closed without merging**